web-library-build/src/index.ts

```diff
@@ -34,30 +34,33 @@ const sourceMatch: string[] = [
 ];
 
 // Define default task groups.
-let buildTasks: IExecutable = task('build', serial(preCopy, sass, parallel(tslint, typescript, text), postCopy));
```

Comment:
they allow you to specify a map of dest: [sources] to copy from one place to another. Sometimes you need that to be done before all the building, sometimes after.